argoverse/utils/sim2.py

```diff
@@ -98,6 +98,8 @@ def transform_from(self, point_cloud: np.ndarray) -> np.ndarray:
         Returns:
             transformed_point_cloud: Nx2 array representing 2d points in frame B
         """
+        if not point_cloud.ndim == 2:
```

Comment:
What is the motivation for overriding the default broadcasting behavior in `numpy`?

Reply:
I guess the main reason here is that a (2,) vector and a (1,2) vector will have very different results, and only the (1,2) case will be correct, in the way we've defined it.

Reply:
if we have a (2,2) array, we also have to be careful to make sure which direction we broadcast the sum with the translation over

Reply:
Thanks for working on this, @johnwlambert. I was wondering what you thought about the following:

```
point_cloud = np.atleast_2d(point_cloud)
```

and then we could simplify by removing lines 101 - 103. Thoughts?